pkg/ebpftracer/c/tracee.bpf.c

```diff
@@ -1853,12 +1890,25 @@ statfunc u32 cgroup_skb_generic(struct __sk_buff *ctx, enum flow_direction flow_
     if (!sk)
         return 1;
 
-    net_task_context_t *netctx = bpf_sk_storage_get(&net_taskctx_map, sk, 0, 0); // obtain event context
+    // TODO(patrick.pichler): fallback to use bpf_skb_cgroup_id to get cgroupv2 and attribute
+    // traffic to cgroup (which we can turn into a container) to not loose it.
+    net_task_context_t *netctx =
+        bpf_sk_storage_get(&net_taskctx_map, sk, 0, 0); // obtain event context
     if (!netctx) {
-        // 1. kthreads receiving ICMP and ICMPv6 (e.g dest unreach)
-        // 2. tasks not being traced (socket was created before agent started)
-        // ...
-        return 1;
+        struct sock *key = (void *) ctx->sk;
```

Comment:
Check for `LINUX_KERNEL_VERSION < KERNEL_VERSION(5, 11, 0)` is missing here, no need to lookup it for new versions